implementation/configuration/src/configuration_impl.cpp

```diff
@@ -342,7 +342,7 @@ bool configuration_impl::load(const std::string &_name) {
 
     std::chrono::steady_clock::time_point end = std::chrono::steady_clock::now();
 
-    for (auto i : its_input) {
+    for (auto const& i : its_input) {
```

Comment:
Prefer `const auto` over `auto const`.

```suggestion
    for (const auto& i : its_input) {
```